src/SIM/SIM2D.C

```diff
@@ -78,6 +78,10 @@ bool SIM2D::addConnection (int master, int slave, int mIdx,
     return false;
   }
 
+  adm.dd.ghostConnections.insert(DomainDecomposition::Interface{master, slave,
+                                                                mIdx, sIdx, orient,
+                                                                dim, basis, thick});
+
```

Comment:
jeg ser ikke noen grunn til å opprettholde feildesignet, altså å kaste bort topologi-informasjonen. hvorfor videreføre dette? jeg er fullstendig åpen for å flytte denne informasjonen ut av dd -klassen hvis det er det du har problemer med.